src/Avalonia.Native/WindowImplBase.cs

```diff
@@ -16,6 +16,34 @@
 
 namespace Avalonia.Native
 {
+    public class MacOSTopLevelWindowHandle : IPlatformHandle, IMacOSTopLevelPlatformHandle
+    {
+        IAvnWindowBase _native;
+
+        public MacOSTopLevelWindowHandle(IAvnWindowBase native)
+        {
+            _native = native;
+        }
+
+        public IntPtr Handle => IntPtr.Zero;
```

Comment:
It should probably point to `NSWindow` property with `"NSWindow"` as `HandleDescriptor`

Reply:
fixed